### PR TITLE
Removing intermediate classes from the drivers. Fixes #135

### LINKED ISSUE #135: Remove Intermediate Classes
Some classes use intermediate classes which driver then inherits from instead of using the base classes themselves. These classes should be removed to keep the hierarchy as flat as possible.

### PR DESCRIPTION
**Pull request by SgStapleton**
The git comparison for a lot of these files is hard to follow. I apologize - not sure how to fix this or if a fix even exists...

### CODE REVIEW

File: ale/drivers/mro_drivers.py [hunk 1]
```diff
@@ -109,13 +127,22 @@ def instrument_id(self):
         : str
           instrument id
         """
-        return self.id_lookup[self.label['INSTRUMENT_NAME']]
+        id_lookup = {
```
Comment (SgStapleton) [inline ale/drivers/mro_drivers.py]: I went ahead and threw this straight into the only property that makes use of it. This should be fine so long as no one else needs to access this value for whatever reason. 

I did also not include it in the preceding property as it did not make use of it. I hope this was ok as well, but someone check my logic.

**Comment by jessemapel** [comment]
CI is done and passed on this, it looks like Github just didn't get the message back from Travis

File: ale/drivers/messenger_drivers.py [hunk 2]
```diff
@@ -164,3 +182,59 @@ def starting_ephemeris_time(self):
             sclock = self.label['IsisCube']['Archive']['SpacecraftClockStartCount']
             self._starting_ephemeris_time = spice.scs2e(self.spacecraft_id, sclock)
         return self._starting_ephemeris_time
+
+    @property
+    def _focal_length(self):
```
Comment (jessemapel) [inline ale/drivers/messenger_drivers.py]: Why is the focal length private?

File: ale/drivers/messenger_drivers.py [hunk 3]
```diff
@@ -164,3 +182,59 @@ def starting_ephemeris_time(self):
...
+
+        Returns
+        -------
+        : string
+          Path to latest metakernel file
+        """
+        return int(spice.gdpool('INS{}_FPUBIN_START_LINE'.format(self.ikid), 0, 1)[0])
+
+    @property
+    def _detector_center_sample(self):
```
Comment (jessemapel) [inline ale/drivers/messenger_drivers.py]: Same as focal length, why private?

File: ale/drivers/messenger_drivers.py [hunk 4]
```diff
@@ -164,3 +182,59 @@ def starting_ephemeris_time(self):
...
+        return float(spice.gdpool('INS{}_BORESIGHT'.format(self.ikid), 0, 3)[0])
+
+
+    @property
+    def _detector_center_line(self):
```
Comment (jessemapel) [inline ale/drivers/messenger_drivers.py]: Again why private?

File: ale/drivers/dawn_drivers.py [hunk 5]
```diff
@@ -152,3 +123,27 @@ def starting_ephemeris_time(self):
             self._starting_ephemeris_time = spice.scs2e(self.spacecraft_id, sclock)
             self._starting_ephemeris_time += 193.0 / 1000.0
         return self._starting_ephemeris_time
+
+    @property
+    def _odtk(self):
```
Comment (SgStapleton) [inline ale/drivers/dawn_drivers.py]: @jessemapel I imagine this needs to be public as well?

File: ale/drivers/messenger_drivers.py (hunk 2 above)
Comment (SgStapleton) [reply ale/drivers/messenger_drivers.py]: I quite literally rearranged code that already existed, so I did not think to check signatures...

Comment (SgStapleton) [reply ale/drivers/messenger_drivers.py]: _focal_plane_temperature above is also private, but it is only used within focal_length. Do I need to make this public for testing purposes, or is it fine to simply test focal_length?

File: ale/drivers/dawn_drivers.py (hunk 5 above)
Comment (jessemapel) [reply ale/drivers/dawn_drivers.py]: This is not part of the base Driver interface so it can be private. If you look at #144 it accesses this via `_odtk`

File: ale/drivers/messenger_drivers.py (hunk 2 above)
Comment (jessemapel) [reply ale/drivers/messenger_drivers.py]: you can test private things in Python, the `_` for private is only a suggestion.
